qt/widgets/common/inc/MantidQtWidgets/Common/FunctionModel.h

```diff
@@ -61,6 +61,7 @@ class EXPORT_OPT_MANTIDQT_COMMON FunctionModel : public IFunctionModel {
                               double value) override;
   void setLocalParameterValue(const QString &parName, int i, double value,
                               double error) override;
+  void setGlobalParameterValue(const QString &paramName, double value);
```

Comment:
I wonder whether this needs to be a public member of the class. Also, unlike almost every other method in this file, this isn't overriding a base class method. I think if `setLocalParameterValue` is part of the interface in the base class then `setGlobalParameterValue` should probably also be in the base class.

Reply:
Good point 👍